userguide/system.rst

```diff
@@ -2834,6 +2834,10 @@ in standby mode. The |web-ui| shows an additional
 :guilabel:`Force Takeover` button which can be used to force that node
 to take control.
 
+Failover is non-functional when both nodes have an identical CARP
```

Comment:
Didn't we fix this wording earlier in another PR?  Being specific about whether failover just will not work or is not allowed, clarifying that it's not that they have an identical CARP state, but the *same* CARP state...